salt/serializers/configparser.py

```diff
@@ -21,18 +21,37 @@
 available = True
 
 
+def _check_preservecase_option(**options):
+    '''
+    Checks options passed to the serialize/deserialze function
+    for the 'casesensitive' argument.  If passed as True,
+    the configparser object will preserve case in items
+    '''
+    preserve_case = False
+    if 'preserve_case' in options:
+        if options['preserve_case']:
+            preserve_case = True
+        options.pop('preserve_case')
+    return options, preserve_case
+
+
 def deserialize(stream_or_string, **options):
     '''
     Deserialize any string or stream like object into a Python data structure.
 
     :param stream_or_string: stream or string to deserialize.
     :param options: options given to lower configparser module.
     '''
+    options, PRESERVE_CASE = _check_preservecase_option(**options)
 
     if six.PY3:
```

Comment:
you should create a helper function that gets the config parser set up correctly, and use that from both locations.